meta.yaml

```diff
@@ -0,0 +1,41 @@
+
+{% set name = "FlaGs" %}
+{% set version = "1.2.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/chayan7/FlaGs/archive/v1.2.6.tar.gz
+  sha256: 0d64fa8a24ad1d79b9ea570549bf2278cd899ab15f65f63b5b65d7420a229f7d
+
+build:
+  noarch: python <=3.6
+  number: 0
+
+
+requirements:
+  build:
+  host:
+    - python <=3.6
```

Comment:
conda-forge does not currently support python <3.6 in `staged-recipes`. There are ways of supporting 2.7 once it gets to the feedstock repo.